<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QMainWindow" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1250</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1250</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="dialogDisplay">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>671</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Times New Roman&quot;;
background-color: rgb(171, 171, 171);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="characterName">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="positiveDialog">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>530</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="negativeDialog">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>530</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>300</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="interviewMarina">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>370</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string> Marina Oswald</string>
    </property>
   </widget>
   <widget class="QPushButton" name="interviewCop">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>370</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Dallas Cop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="interviewLee">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>370</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string> Lee Harvey Oswald</string>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>761</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(171, 171, 171);
font: 24pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="briefcase">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>850</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
image : url(:/Resources/detectiveBriefcase.png);
background-color: rgba(255, 255, 255, 0);
}

QPushButton:hover{
image : url(:/Resources/openBriefCase.png);
background-color: rgba(255, 255, 255, 0);
}

QPushButton:pressed{
image : url(:/Resources/openBriefCase.png);
background-color: rgba(255, 255, 255, 0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="signalUpdate">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>820</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Resources/update.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="interviewNotes">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>370</y>
      <width>461</width>
      <height>551</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 249, 185);
font: 13pt &quot;Segoe Script&quot;;
border-color: rgb(250, 255, 221);
padding-left:10; padding-top:10; padding-bottom:10; padding-right:10;
text-align:top center;




</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="caseFrame">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>830</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dialogFrame">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>1041</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 127);
background-color: rgb(171, 171, 171);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="caseInfo">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>780</y>
      <width>321</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Times New Roman&quot;;
color: white; background-color: gray;</string>
    </property>
    <property name="text">
     <string>If you learn something new your case notes will be updated</string>
    </property>
   </widget>
   <widget class="QLabel" name="speakerImage">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>caseFrame</zorder>
   <zorder>dialogFrame</zorder>
   <zorder>dialogDisplay</zorder>
   <zorder>characterName</zorder>
   <zorder>positiveDialog</zorder>
   <zorder>negativeDialog</zorder>
   <zorder>nextButton</zorder>
   <zorder>interviewMarina</zorder>
   <zorder>interviewCop</zorder>
   <zorder>interviewLee</zorder>
   <zorder>infoLabel</zorder>
   <zorder>briefcase</zorder>
   <zorder>signalUpdate</zorder>
   <zorder>caseInfo</zorder>
   <zorder>speakerImage</zorder>
   <zorder>interviewNotes</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1250</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
